prov/gni/src/gnix_dom.c

```diff
@@ -427,8 +433,8 @@ gnix_domain_ops_open(struct fid *fid, const char *ops_name, uint64_t flags,
 	return ret;
 }
 
-int gnix_domain_open(struct fid_fabric *fabric, struct fi_info *info,
-		     struct fid_domain **dom, void *context)
+DIRECT_FN int gnix_domain_open(struct fid_fabric *fabric, struct fi_info *info,
```

Comment:
Not exactly.  Some are assigned to the function table in the same file in which they are defined, and those should be STATIC.  Those that are assigned to the function table in a separate file (e.g., fabric.c) should not be.  Again, sorry to be add confusion.